Infrastructure.DataAccess/Migrations/SQLScripts/Migrate_Contract_Relations.sql

```diff
@@ -0,0 +1,84 @@
+﻿/*
+User story reference: 
+    https://os2web.atlassian.net/browse/KITOSUDV-499
+ 
+Content:
+ Migrates existing contract relations according to the following logic
+
+ If an ItInterfaceUse points to an ItInterface for which the exposing system IS IN USE in the organization 
+    AND that system is not the same as the using system, 
+    THEN the interface useage is migrated to a system relation 
+    between the "Using" system usage and the local usage of the exposing system. If data is migrated, the old data is marked with a reference to the new rows in the SystemRelations table.
+*/
+
+BEGIN
+    -- Build an operation context - the from-to context needed to migrate the relation
+    DECLARE @MigrationContext TABLE
```

Comment:
Giver det en praktisk forskel at anvende @table variables frem for #temporary tables?

Reply:
![image](https://user-images.githubusercontent.com/13116889/74820463-59366b00-5302-11ea-8597-60ce3bfbc262.png)

Reply:
Som udgangspunkt ikke noget der bekymrer mig for jeg har ikke brug for de ting der er begrænsninger ved table vars

Reply:
Behøver vi ikke en transaktion?

Reply:
Kun ift. kildetabellen. Den brokker sig i hvert fald ikke når den kører på